File: Bilberry.java
import javax.swing.*;

public class Bilberry extends Plant{
    Bilberry(World world1, int x, int y) {
        SetStrength(0);
        SetInitiative(0);
        SetCords(x, y);
        SetMove(true);
        world = world1;
    }
    @Override
    public String getName() {
        return "BILBERRY";
    }
    @Override
    public void Drawing() {
        ImageIcon icon = new ImageIcon("src/Icons/bilberry.png");
        world.GetButton(GetCords()).setIcon(icon);
    }
    @Override
    public void Collision(Organism attacked) {
        world.GetOrganisms().remove(attacked,this);
        world.GetOrganisms().remove(this,attacked);
        //!!!NOTIFICATIONS WHEN ANIMAL IS DELETED
    }
}

File: bilberry.py
from plant import Plant

class Bilberry(Plant):
    def __init__(self, world1, x, y):
        super().__init__()
        self.SetStrength(0)
        self.SetInitiative(0)
        self.SetCords(x, y)
        self.SetMove(True)
        self.world = world1

    def getName(self):
        return "BILBERRY"

    def Drawing(self):
        self.world.buttons[self.GetCords().y -1][self.GetCords().x -1].configure(bg='#00FF00')
    
    def New(self, text, reference, x, y):
        self.world.GetOrganisms().add(Bilberry(reference, x, y))    

    def Collision(self, attacked):
        self.world.GetOrganisms().remove(attacked, self)
        self.world.GetOrganisms().remove(self, attacked)
        # !!!NOTIFICATIONS WHEN ANIMAL IS DELETED
